spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubEmulatorConfiguration.java

```diff
@@ -36,7 +36,7 @@
  * @author Andreas Berger
  */
 @Configuration
-@ConditionalOnProperty(prefix = "spring.cloud.gcp.pubsub", name = "emulatorHost")
+@ConditionalOnProperty(prefix = "spring.cloud.gcp.pubsub", name = "emulator-host")
```

Comment:
Could you please make the change everywhere else in documentation, etc.?

Reply:
That's good catch, indeed. However you still can use it as `spring.cloud.gcp.pubsub.emulatorHost`. Therefore might be no need to change docs everywhere else.